src/core/components/auth/auths.jsx

```diff
@@ -43,24 +43,21 @@ export default class Auths extends React.Component {
   }
 
   render() {
-    let { definitions, getComponent, authSelectors, errSelectors } = this.props
+    let { definitions, getComponent, authSelectors, errSelectors, getSystem } = this.props
     const ApiKeyAuth = getComponent("apiKeyAuth")
     const BasicAuth = getComponent("basicAuth")
-    const BasicJwtAuth = getComponent("basicJwtAuth", true)
```

Comment:
Unless you want to clean up all related code, e.g. `basic-jwt-auth.jsx` etc etc, please put it back so that there's no dangling code. There are other auth methods, e.g. oauth, apikey, basic auth etc, too. We have no plan to remove / clean up. So please keep them intact.